pontoon/pretranslation/pretranslate.py

```diff
@@ -57,6 +57,15 @@ def get_pretranslations(entity, locale, preserve_placeables=False):
             return []
 
         pretranslation = serializer.serialize_entry(entry)
+        print(pretranslation)
+        # error-not-subscribed = Esta dirección de correo electrónico no está suscrita a {-product-name}.
+
+        # Parse and serialize pretranslation again in order to assure cannonical style
+        parsed_pretranslation = parser.parse_entry(pretranslation)
```

Comment:
The serializer doesn't check for junk at its `serialize_entry()` enpoint, so this should be included:

```suggestion
        parsed_pretranslation = parser.parse_entry(pretranslation)
        if isinstance(parsed_pretranslation, FTL.Junk):
            log.info(f"Fluent pretranslation error: Invalid translation: {pretranslation}")
            return []
```

Also need a new import at the top of the file:
```python
from fluent.syntax import ast as FTL
```